packages/web-components/src/utils/types.ts

```diff
@@ -28,6 +28,8 @@ export enum IcInformationStatus {
   Success = "success",
 }
 
+export type IcSizes = "default" | "large" | "small";
```

Comment:
I believe this type is also used in IcButton aswell. Would be worth updating it there as well